sconstruct

```diff
@@ -125,7 +125,7 @@ env = Environment(variables=vars,HOST_ARCH='x86',tools=[
 # Specifically request to use Visual Studio 2019 (MSVC 14.2).
 # This will ensure that this specific version is selected if there are multiple versions installed,
 # And fail the build if this version is not installed at all.
-env.SetDefault(MSVC_VERSION='14.2')
+#env.SetDefault(MSVC_VERSION='14.2')

```

Comment:
Please delete this line and the comment above.
But, we should also reintroduce code into nvdaHelper/archBuild_sconscript to ensure the chosen msvc version is 14.2 or higher.